### DISCUSSION TITLE
findMany with multiple values

### DISCUSSION THREAD

**Discussion by gsmatheus** [prisma/prisma#14596] [Q&A] [answered]
How can I use the prisma to have the same result as this query ?

`SELECT * FROM products WHERE product.idProduct IN (332,2,3)`

**Comment by austincrim** [comment, marked as answer]
Hey @gsmatheus 👋,

Depending on your `schema.prisma`, it would look something like this:
```ts
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()

const result = await prisma.product.findMany({
  where: {
    idProduct: { in: [332, 2, 3] }
  }
})
```

You can see more examples of `in` [here](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#in).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
